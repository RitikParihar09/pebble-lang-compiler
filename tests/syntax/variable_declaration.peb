// Test: Variable declaration syntax
// Testing different ways to declare variables

// Integer
var num = 42
display(num)

// Float
var pi = 3.14159
display(pi)

// String
var greeting = "Hello, World!"
display(greeting)

// Boolean
var is_active = true
display(is_active)

// Character
var letter = 'A'
display(letter)

// Constants
const gravity = 9.8
display(gravity)
