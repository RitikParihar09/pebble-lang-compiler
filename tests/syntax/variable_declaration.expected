42
3.14159
Hello, World!
1
A
9.8
